Start of object browser (#26)

worksofliam opened the pull request:
### Changes

Covers issue #26. This adds an object browser to the IBM i view. I am using the same caching technique used in the member browser by @barrettotte.

Right-click contexts include:

* Run Action

### Checklist

* [X] have tested my change
* [x] updated relevant documentation
* [X] Remove any/all `console.log`s I added
* [X] eslint is not complaining
* [X] have added myself to the contributors' list in the README

worksofliam commented:
@barrettotte I have given you 'read' access on a repo that says you can now review PRs. Any chance, whenever you have the time AND if you wanted, you could review this PR for me?

If not, let me know. 

Thanks :)

worksofliam commented:
Before this is merged, should I add more right-click options?

* Delete
* Change text?

barrettotte commented on src/views/objectBrowser.js:
```diff
@@ -0,0 +1,148 @@
+  'FILE': 'database',
+  'CMD': 'terminal',
+  'MODULE': 'extensions',
+  'PGM': 'chevron-right',
```
Is there eventually going to be nested info in objects of type *PGM? If not, maybe setting the icon to something like ```'PGM': 'file-binary'``` would make sense. I really like the other icons though, it feels so natural

worksofliam replied:
`file-binary` is much nicer. I will change that.

barrettotte reviewed (commented):
Code looks awesome and follows the same pattern as the other browsers. I also like the use of the targeted refresh by library. The icons feel very natural minus *PGM, let me know what you think of the comment I left on objectBrowser.js

One thing that I thought of while testing a couple small CL compiles:  Is there any easy way to refresh the object browser after running a compile? That way the newly created PGM would show up in the object list. If not, its not a big deal at all to just refresh

worksofliam commented:
@barrettotte 

I have fixed an issue where the right-click refresh was not showing up. I also changed the icon to `file-binary`. As well as those, if you have auto-refresh enabled, it will invoke`refreshObjectList` after an action has finished executing.

Please review again :)

barrettotte reviewed (approved):
Just tried compiling again and the appropriate library refreshed on compile, thanks!

worksofliam commented:
Will be merging today! Thanks @barrettotte :)
